docs/_templates_overwrite/index.html

```diff
@@ -1,568 +1,628 @@
+                            <p>The language is a superset of Python 3.5+ with additional shell primitives that you are used to from Bash and IPython.</p>
                         </div>
-                        <div class="col-lg-12">
-                            <div class="single-gallery">
-                                <div class="overlay"></div>
-                                <img class="img-fluid w-100" src="_static/xonsh-docker-example.jpg" alt="">
-                                <div class="content">
-                                    <a class="pop-up-image" href="_static/xonsh-docker-example.jpg">
-                                        <i class="lnr lnr-eye"></i>
-                                    </a>
-                                </div>
-                            </div>
+                    </div>
+                </div>
+
+                <div class="col-md-6">
+                    <div class="features-single">
+                        <div class="icon">
+                            <i class="icofont-light-bulb"></i>
+                        </div>
+                        <div class="content">
+                            <h4>Xontribs - the 3rd-party extension system.</h4>
+                            <p>Xontributions, or xontribs, are a set of tools and conventions for extending the functionality of xonsh beyond what is provided by default.</p>
+                        </div>
+                    </div>
+                </div>
+
+                <div class="col-md-6">
+                    <div class="features-single">
+                        <div class="icon">
+                            <i class="icofont-history"></i>
+                        </div>
+                        <div class="content">
+                            <h4>Rich interface to discover history</h4>
+                            <p>The xonsh shell implements a collection of history files that have rich structure and API for discover them.</p>
                         </div>
                     </div>
                 </div>
-                <div class="col-lg-4 hidden-md hidden-sm">
-                    <div class="single-gallery">
-                        <div class="overlay"></div>
-                        <img class="img-fluid w-100" src="_static/listunison.png" alt="">
+
+                <div class="col-md-6">
+                    <div class="features-single">
+                        <div class="icon">
+                            <i class="icofont-fix-tools"></i>
+                        </div>
                         <div class="content">
-                            <a class="pop-up-image" href="_static/listunison.png">
-                                <i class="lnr lnr-eye"></i>
-                            </a>
+                            <h4>Powerful prompt customisation</h4>
+                            <p>Customisable tab completion, key bindings, color styles are on board by default.</p>
```

Comment:
```suggestion
                            <h4>Powerful prompt customization</h4>
                            <p>Customizable tab completion, key bindings, color styles are included by default.</p>
```
The rest of our docs use north american english, so to be consistent with that these should be `z`s instead of `s`s